akka-distributed-data/src/main/scala/akka/cluster/ddata/ORMultiMap.scala

```diff
@@ -80,11 +98,11 @@ final class ORMultiMap[A, B] private[akka] (private[akka] val underlying: ORMap[
   def getOrElse(key: A, default: ⇒ Set[B]): Set[B] =
     get(key).getOrElse(default)
 
-  def contains(key: A): Boolean = underlying.contains(key)
+  def contains(key: A): Boolean = underlying.keys.elements.contains(key)
```

Comment:
should `contains` in `underlying` be changed instead?
is this only needed for delta-values or is it a good change in general?